### PR TITLE
add suggestions UI to Search

### PR DESCRIPTION
**Pull request by aliceriot**
#### Pre-Flight checklist

- [x] Screenshots and design review for any changes that affect layout or styling
  - [x] Desktop screenshots
  - [x] Mobile width screenshots
  - [x] Tag @ferdi or @pdpinch for review
- [x] Testing
  - [x] Code is tested
  - [x] Changes have been manually tested

#### What are the relevant tickets?

part of #178 

#### What's this PR do?

this adds a basic 'suggestion' UI, along the lines of what we do in Open course search.

#### How should this be manually tested?

Make a typo in the search, like "sceince", "mothemotics", or something similar. you should get a little suggestion for what you mean, and click on it should change to the search text to that suggestion.

#### Screenshots (if appropriate)

Screenshot from 2020-10-29 14-57-45
Screenshot from 2020-10-29 14-55-33

### CODE REVIEW

**Comment by aliceriot** [comment]
@pdpinch @Ferdi @abdulkdawson I couldn't find a mockup for search suggestions, so I just threw something together really quick. lmk if this is close or not!

**Review by mbertrand — APPROVED** [review]
👍

**Comment by aliceriot** [comment]
@mbertrand thanks! waiting on a design sign-off now

**Comment by Ferdi** [comment]
@aliceriot it's good to merge this as if for now

**Pull request merged**
